### DISCUSSION TITLE
Separate server or same server for OnlyOffice DocumentServer

### DISCUSSION THREAD

**Discussion by webdesign7** [ONLYOFFICE/DocumentServer#2601] [Q&A]
I need an advice / best practice on installing OnlyOffice DocumentServer 

Currently I have NextCloud on one server ( RedHat ) so i am thinking whether to install OnyyOffice Document Server on same server or within same server. ? 

How is it recommended separate instance or same instance  ?  I'd like to know some reasons as well please 

Thank you

**Comment by igwyd** [comment]
Hello, it depends on your requirements, how many users?

**Reply by webdesign7** [reply]
Hello,  up to 200 users but the number will grow

**Reply by igwyd** [reply]
I don’t have data on the resource consumption of the nextcloud server, so I can’t say for sure which server needed for this. OnlyOffice DocumentServer - up to 500-600 concurrent users I would recommend the server 2 CPU and 4 ram, this is confirmed by the results of our load tests.
